gbpservice/nfp/configurator/drivers/loadbalancer/v2/haproxy/octavia_lib/amphorae/driver_exceptions/exceptions.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2011-2014 OpenStack Foundation,author: Min Wang,German Eichberger
+#
```

Comment:
Ignored review of all the Octavia lib code as it is a copy from Octavia code base.